r2-shared/Publication/Metadata.ts

```diff
@@ -0,0 +1,128 @@
+import { IContributor} from "./Contributor";
+import { ISubject } from "./Subject";
+import { LocalizedString } from "./LocalizedString";
+import { ReadingProgression } from "./ReadingProgression";
+import Presentation from "./presentation/Presentation";
+
+interface Collection extends IContributor {
```

Comment:
Yeah I wish we can come up with best practices on when to use interfaces, type aliases, etc. and their syntax. 

Lots of people have been used to using interfaces by default because [you could not extend nor implement from type aliases prior to TS 2.7](https://www.typescriptlang.org/docs/handbook/advanced-types.html#interfaces-vs-type-aliases) so it can quickly become a mess if we don’t set guidelines.